site/src/pages/spec/[[...specSlug]].page.tsx

```diff
@@ -270,12 +268,12 @@ const SpecPage: NextPage<SpecPageProps> = ({ serializedPage }) => {
           sx={{
             marginLeft: {
               xs: 0,
-              md: "300px",
+              md: "280px",
             },
             maxWidth: {
               xs: "100%",
               sm: `calc(100% - ${INFO_CARD_WIDTH}px)`,
-              md: `calc(100% - ${INFO_CARD_WIDTH}px - 300px)`,
+              md: `calc(100% - ${INFO_CARD_WIDTH}px - 280px)`,
```

Comment:
Yes, please do - mentally bookmarked that we should fix that, but wasn't sure how

Reply:
Attempted to fix this on my recent commit https://github.com/blockprotocol/blockprotocol/pull/67/commits/23b45e2142849550855cc29b803a762093c7d13a, but when you resize the browser window the 'next' link moves way over into the middle of the space. Not sure how to fix it without refactoring and setting the container to `justify-items: space-between`

Reply:
I think I managed to fix this in this in https://github.com/blockprotocol/blockprotocol/pull/67/commits/379cf8f54784a16c833e2a96df3c3e26f37a824b (in addition to this I added a custom breakpoint where the next/prev arrows would be hidden to avoid them from overflowing the page).